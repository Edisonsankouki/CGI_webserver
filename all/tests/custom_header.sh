cd ..
python3 webserv.py config.cfg &
PID=$!
cd -
sleep 1
str=$(curl -I localhost:8070/hello_1.py | diff - custom_header.out)

if [[ "$str" == "" ]];then
    echo test_custom passed
else
    echo test_custom passed
fi 
kill $PID