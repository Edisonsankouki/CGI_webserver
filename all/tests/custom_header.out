HTTP/1.1 203 hahaha
